cmd/template/dbdriver/files/tests/scylla.tmpl

```diff
@@ -0,0 +1,137 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"github.com/docker/go-connections/nat"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+	"io"
+	"log"
+	"strings"
+	"testing"
+)
+
+const (
+	port = nat.Port("19042/tcp")
+)
+
+func mustStartScyllaDBContainer() (testcontainers.Container, error) {
+
+	// Define the container
+	name := "scylla-test"
+	image := "scylladb/scylla:6.2"
+	exposedPorts := []string{"9042/tcp", "19042/tcp"}
+	commands := []string{
+		"--smp=2",
+		"--memory=1G",
+		"--developer-mode=1",
+		"--overprovisioned=1",
+	}
+
+	req := testcontainers.ContainerRequest{
+		FromDockerfile: testcontainers.FromDockerfile{},
+		Image:          image,
+		ExposedPorts:   exposedPorts,
+		Cmd:            commands,
+		WaitingFor: wait.ForAll(
+			wait.ForLog(".*initialization completed.").AsRegexp(),
+			wait.ForListeningPort(port),
+			wait.ForExec([]string{"cqlsh", "-e", "SELECT bootstrapped FROM system.local"}).WithResponseMatcher(func(body io.Reader) bool {
+				data, _ := io.ReadAll(body)
+				return strings.Contains(string(data), "COMPLETED")
+			}),
+		),
+		Name: name,
+	}
+
+	// Start the container
+	scyllaDBContainer, err := testcontainers.GenericContainer(
+		context.Background(), testcontainers.GenericContainerRequest{
+			ContainerRequest: req,
+			Started:          true,
+		})
+	if err != nil {
+		return nil, err
+	}
+
+	mappedPort, err := scyllaDBContainer.MappedPort(context.Background(), "19042/tcp")
+	if err != nil {
+		return nil, err
+	}
+
+	hosts = fmt.Sprintf("localhost:%v", mappedPort.Port())
```

Comment:
bug: we'd need to initialise the rest of the fields (username, password, consistencyLevel) in order to bootstrap the cluster correctly. Please check how to define the credentials in the docker image, so that the container request sets them.

Reply:
@DanielHe4rt would you be interested in contributing a ScyllaDB module for testcontainers-go? This way here we'd use the module instead of the GenericContainer approach. In the end is the same, but easier for the end user to interact with a built-in module.

I can help you out with the creation of the module. Thoughts?

Reply:
I'm actually with a task to work on it under the Testcontainers with Go :p 

Quite amazing to have a core-contributor looking for new contributions around! We can chat and make this a thing, even start some Livestream while we work on it.

I just sent you a LinkedIn connection, we can chat there if you want!

Reply:
> bug: we'd need to initialise the rest of the fields (username, password, consistencyLevel) in order to bootstrap the cluster correctly. Please check how to define the credentials in the docker image, so that the container request sets them.

@DanielHe4rt Was there a particular reason for not including this in the fix?

Reply:
Sorry, I'm not a Scylla user, so I wrongly understood the credentials needed to be set. LGTM then